src/game/behaviors/moat_drainer.inc.c

```diff
@@ -1,8 +1,12 @@
 // moat_drainer.inc.c
 
 void bhv_invisible_objects_under_bridge_init(void) {
+#ifndef UNLOCK_ALL
     if (save_file_get_flags() & SAVE_FLAG_MOAT_DRAINED) {
+#endif
         gEnvironmentRegions[6] = -800;
         gEnvironmentRegions[12] = -800;
+#ifndef UNLOCK_ALL
     }
+#endif
```

Comment:
not a huge deal but you could always have the if statement check true and it should get optimized out, then you dont have to use two ifdefs

Reply:
non blocking, i dont think its a huge deal